Orange/widgets/data/utils/preprocess.py

```diff
@@ -412,7 +413,7 @@ def createWidgetFor(self, index):
         Create a QWidget instance for the index (:class:`QModelIndex`)
         """
         definition = index.data(DescriptionRole)
-        widget = definition.viewclass()
+        widget = definition.viewclass(parent=self._parent)
```

Comment:
Why? The `self._parent` is not even the correct type (it can be any QObject, but QWidgets must be parented to other QWidgets), and has no particular specified relation to the parent-child hierarchy that the widgets are/will be in.

Reply:
I need a reference to OWWidget. I should probably have used some other keyword argument.

Reply:
If you use any other keyword argument, you are, I think, going to break the existing preprocessors.

I do not know what is a good solution here. You could perhaps use `widget.parent_widget = self._parent` after init. At least that is what I did when I subclassed the Controller.